### ISSUE TITLE
PHP 8 Support?

### ISSUE THREAD

**Issue by harizinside** [kavist/rajaongkir#19]
some problem after upgrade to php 8

```consolde
Problem 1
    - Root composer.json requires kavist/rajaongkir ^1.1.0 -> satisfiable by kavist/rajaongkir[v1.1.0].
    - kavist/rajaongkir v1.1.0 requires php ^7.0 -> your php version (8.0.1) does not satisfy that requirement.
```

is it still get any update?

**Comment by irhamumam17** [comment]
same problem
[edited]
composer require kavist/rajaongkir:^1.1 --ignore-platform-reqs

**Comment by harizinside** [comment]
> same problem
> [edited]
> composer require kavist/rajaongkir:^1.1 --ignore-platform-reqs

Yeah it's solve problem, but i'm no longer using this package. Seems the package has abandoned

**Comment by 26kito** [comment]
> same problem [edited] composer require kavist/rajaongkir:^1.1 --ignore-platform-reqs

Thank you, it works. Anyway is peoples still use this package? Seems like it's outdated
